History.md

```diff
@@ -1,6 +1,15 @@
 unreleased
 ==========
 
+  * deps: statuses@2.0.0
```

Comment:
The `deps` changes should be sorted to the end of the list (see previous releases for example).